draft-ietf-lamps-kyber-certificates.md

```diff
@@ -549,87 +549,86 @@ key in the previous section.
+         :     53 0F 8C B6 38 A8 1A 62 AC 30 18 49 AB AF 95 A7
+         :     30 1B DA 30 06 89 09 BF DB 7E 67 DB CC BB 38 A5
+         :     55 1A 25 B1 A3 A0 F6 85 74 8A D5 75 3D 88 80 F0
+         :     01 6C 62 74 86 16 63 84 C5 57 1F E2 36 59 00 36
+         :     4D 03 83 11 E2 D8 75 DB 36 66 86 93 2B 5E C6 02
+         :     43 0A 36 9E 87 A6 EF 5C 33 87 86 65 78 25 BD 4C
+         :     05 7A CE B9 23 EB 09 35 E6 90 5E 63 B4 CE D7 F8
+         :     08 57 A7 73 DD 64 B1 50 D2 66 12 EA 9A C1 20 52
+         :     DB 20 17 BF 18 43 CC B4 B3 28 1B 69 0D C7 28 AD
+         :     FA 85 C0 02 81 B8 E3 C0 92 87 33 5F 85 6B 4F C2
+         :     89 2F 69 A2 F5 79 21 AD A0 19 14 C4 09 88 66 2D
+         :     57 76 96 62 A7 86 35 1B 9B 66 49 3D AB 79 59 4D
+         :     98 6D E2 10 0D 65 BA 0F F4 EA 58 B8 15 38 D2 4A
+         :     44 35 A2 58 FA C2 54 04 AA 7F 41 F6 58 B1 38 50
+         :     65 E1 58 DC B6 01 15 73 27 20 F4 04 59 AA AC 15
+         :     E4 06 95 3A 90 AC 52 99 7D 1C CD 07 00 60 EF C6
+         :     5D B9 E6 53 35 44 67 FA D5 6E C7 13 C8 6E 75 40
+         :     C4 23 AC F2 66 9F 52 FA 6F 4A C6 88 8D 87 1E F3
+         :     E8 47 C0 29 A8 AA FB B9 2E 17 B2 4A A0 79 B1 F4
+         :     19 BA 61 75 B4 42 AF B1 19 09 D4 A5 6B 70 A0 33
+         :     5B 28 73 92 18 AA 7C 93 48 E2 C3 C2 F3 EB 3D 15
+         :     A4 1E 64 17 C0 DD 94 BF EB 21 41 9B 31 1A 7B B1
+         :     3A 18 0B BE 83 32 18 A9 A6 B1 74 47 CC 85 F2 25
+         :     85 95 87 A7 30 77 04 9A CB CF D4 4D 0F 02 54 38
+         :     E1 5D 15 38 27 0D 58 6E 1B F8 31 92 A9 45 9C F6
+         :     3C 0E 97 2F 85 29 76 79 83 1E CF 12 15 09 85 1C
+         :     B8 34 0F 6F 10 7B 0F A1 A0 EF D1 B3 6A 81 89 BC
+         :     08 5C 4F 5C B7 84 E5 53 F4 1B 91 8F 80 39 7C E1
+         :     95 6F 78 5B EE 37 7C A9 AA 8B E6 99 8A DA 30 C2
+         :     6B 7C 3D 8C 6B 55 25 4C C9 62 03 B2 0C 42 AE E0
+         :     AC 4E 1E BB 40 8E 49 A9 E3 F8 79 D0 AB 07 85 EB
+         :     70 25 42 5D 13 05 A2 29 9C 01 5E 12 0D 16 3B 0E
+         :     19 49 4C E5 72 53 D0 24 6D 18 27 45 CB 81 97 AB
+         :     74 38 B3 C1 BB 79 72 BE C5 A3 06 EB A3 56 78 55
+         :     C0 14 69 9F EF 65 AE 54 C7 70 A0 D8 5C 18 40 0C
+         :     F6 42 AE DC 66 07 77 BA 4B 13 85 02 BD 5A 78 12
+         :     F6 21 F8 4A 48 29 6B 98 DD 43 22 B6 F1 58 28 B8
+         :     A8 F0 E0 0A 8B A4 4A 53 C3 A8 B1 43 57 1B 07 40
+         :     AB D5 67 DA F1 CD E9 C7 9C 20 4B 6D 5E 25 9D 17
+         :     66 A3 1B BB CB 4E 6A 05 CF 45 02 17 6B 30 1C 1C
+         :     2F 41 24 77 50 15 7B CE C8 5E 80 9B 30 A4 D6 0D
+         :     77 47 CD D0 F5 B9 9A A8 C8 26 98 75 17 79 3A AA
+         :     80 80 A0 B1 24 A8 55 8D F7 2B BE 37 B7 5F 4E DB
+         :     B6 BE 82 16 D6 C6 33 FB 2B 22 80 E2 51 13 D8 69
+         :     5E 43 48 1C 3E EB 39 7E B1 92 50 52 29 B6 7A 20
+         :     1E A8 93 C3 E2 CB 32 DA 8B C3 42 FA 4D EA 05 78
+         :   }
 ~~~
 
```

Comment:
```suggestion
<aside markdown="block">
NOTE: The first byte of the BIT STRING's content, the padding byte, has been stripped from the pretty print above.
</aside>

```

Reply:
I don't like calling the padding part of the content because it's not part of the public key.  How about:

NOTE: The padding byte of the DER-encoded BIT STRING is not displayed in the pretty print above.